### DISCUSSION TITLE
how to support multiple cell selection here like react-spreadsheet or any other spreadsheet/excel supports ?

### DISCUSSION THREAD

**Discussion by vaibhav-systango** [Comcast/react-data-grid#3319] [Q&A]
need to be able to select multiple cells by mouse drag or CTRL button

**Comment by vaibhav-systango** [comment]
@nstepien  @malonecj  @amanmahajan7 @diogofcunha

**Reply by msindal** [reply]
Looking for the same feature in my app, please help
